astrolabe/runner.py

```diff
@@ -69,19 +72,10 @@ def cluster_url(self):
     def get_connection_string(self):
         if self.__connection_string is None:
             cluster = self.cluster_url.get().data
-            prefix, suffix = cluster.srvAddress.split("//")
-            uri_options = self.spec.maintenancePlan.uriOptions.copy()
-
-            # Boolean options must be converted to lowercase strings.
-            for key, value in uri_options.items():
-                if isinstance(value, bool):
-                    uri_options[key] = str(value).lower()
-
-            connection_string = (prefix + "//" + self.config.database_username
-                                 + ":" + self.config.database_password + "@"
-                                 + suffix + "/?")
-            connection_string += urlencode(uri_options)
-            self.__connection_string = connection_string
+            uri = re.sub(r'://',
+                '://%s:%s@' % (self.config.database_username, self.config.database_password),
+                cluster.srvAddress)
```

Comment:
Since the workload executors cannot add URI options to the connection string, don't we need to add them to the connection string being generated here? This was previously being done (rather simplistically) using 

```
connection_string += urlencode(uri_options)
```

but now we don't seem to be doing it at all. Can you elaborate on how workload executors are expected to respect URI options if they cannot augment the incoming connection string and if astrolabe doesn't augment the outgoing connection string?